src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -237,7 +237,7 @@ std::size_t ReconstructionEngine_sequentialSfM::fuseMatchesIntoTracks()
     tracksBuilder.build(matches);
 
     ALICEVISION_LOG_DEBUG("Track filtering");
-    tracksBuilder.filter(_params.filterTrackForks, _params.minInputTrackLength);
+    tracksBuilder.filter(true,_params.filterTrackForks, _params.minInputTrackLength);
```

Comment:
This one was correct before but not anymore.
We should probably remove the default values for the 2 first parameters of the filter function.